akka-projection-r2dbc/src/main/resources/reference.conf

```diff
@@ -3,8 +3,6 @@
 
 //#projection-config
 akka.projection.r2dbc {
-  # postgres or yugabyte
-  dialect = ${akka.persistence.r2dbc.dialect}
```

Comment:
Yes, this will work also for the case when using different databases, with different dialects, for the write side and the offset store (read side). `use-connection-factory` is the pointer to the dialect.